projects/m4/014-words-with-six-vowels-in-order/js/read_file.js

```diff
@@ -0,0 +1,19 @@
+const fs = require('fs');
+
+const readFile = file => {
+
+        return new Promise ((resolve, reject) => {
+
+            fs.readFile (`./files/${file}`, 'utf-8', (err, data) => {
+
+                if (err) reject (`ERROR: The requested file does not exist!`);
+                resolve (data);
+                
+            });
+
+        });
+    
+}; 
+
+module.exports = readFile;
+
```

Comment:
To Fix:
- change the **name** of **some functions**
- remove **unnecessary functions**
- use **async/await** instead of _double then_